### PR TITLE
Move /picks to /compat/picks

### PR DESCRIPTION
**Pull request by barthalion**

### CODE REVIEW

File: backend/app/main.py [hunk 1]
```diff
@@ -100,6 +99,11 @@ def update_stats():
     stats.update(apps)
 
 
+@app.post("/update/exceptions")
```
Comment (razzeee) [inline backend/app/main.py:102]: This seems unrelated?

Comment (barthalion) [reply backend/app/main.py:102]: Yeah, some leftover from my local repo. I will split it to separate PR in the evening.

**Review by razzeee — APPROVED** [review]
